recreate_pods.sh

```diff
@@ -1,21 +1,21 @@
 #!/bin/bash
 
-# Delete all pods that are owned by this RC.
-#  - Get the labels that the RC is selecting based on
-#  - Delete all the pods with that set of labels.
-#  - The RC will then recreate the pods.
-#
+# Update the required env vars for the first pod in a Deployments.
+# This will kick off a rolling update.
 # Do this so that the secrets can be remounted.
 
-if [ -z "$RC_NAMES" ]; then
-    echo "WARNING: RC_NAMES not provided. Secret changes may not be reflected."
+if [ -z "$DEPLOYMENTS" ]; then
+    echo "WARNING: DEPLOYMENTS not provided. Secret changes may not be reflected."
     exit
 fi
 
-RC_NAMES=(${RC_NAMES})
+DEPLOYMENTS=(${DEPLOYMENTS})
+DATE=$(date)
```

Comment:
Yeah we don't actually care what this is as long as it's different from the last one. Being consistent is a good thing - but actually date will produce consistent results within this image.
```
$ date
Tue Apr 19 07:56:51 UTC 2016
```